VMEncryption/main/check_util.py

```diff
@@ -99,6 +99,14 @@ def check_kek_url(self, test_kek_url, message):
             raise Exception('\n' + message + '\nActual: ' + test_kek_url + '\nExpected: ' + expected + "\n")
         return
 
+    def check_mhsm_url(self, test_mhsm_url, message):
+        """basic sanity check of the MHSM url"""
+        expected = "https://keyvault-name}.{vault-endpoint}"
```

Comment:
The expected string here needs update. It should refer to managedHSM and not keyvault